lib/rex/proto/mssql/client_mixin.rb

```diff
@@ -32,6 +32,30 @@ module ClientMixin
   STATUS_RESETCONNECTION         = 0x08 # TDS 7.1+
   STATUS_RESETCONNECTIONSKIPTRAN = 0x10 # TDS 7.3+
 
+  # Mappings for ENVCHANGE types
+  # See the TDS Specification here: https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/2b3eb7e5-d43d-4d1b-bf4d-76b9e3afc791
+  ENVCHANGE = {
```

Comment:
https://stackoverflow.com/questions/47153612/best-way-to-define-constants-in-ruby-ror-at-one-place